src/filemanager.cpp

```diff
@@ -1988,3 +1988,34 @@ void pj_set_searchpath(int count, const char **path) {
     proj_context_set_search_paths(nullptr, count,
                                   const_cast<const char *const *>(path));
 }
+
+
+/************************************************************************/
+/*                  proj_context_set_ca_bundle_path()                   */
+/************************************************************************/
+
+/** \brief Sets CA Bundle paths.
+ *
+ * Those CA Bundle path will be used by PROJ when curl and PROJ_NETWORK
+ * are enabled.
+ *
+ * If set on the default context, they will be inherited by contexts created
+ * later.
+ *
+ * The path(s) should be encoded in UTF-8.
+ *
+ * @param ctx PROJ context, or NULL for the default context.
+ * @param path Path. May be NULL.
+ *
+ * @since PROJ 7.2
+ */
+void proj_context_set_ca_bundle_path(PJ_CONTEXT *ctx, const char *path) {
+    if (!ctx)
+        ctx = pj_get_default_ctx();
+    if (!ctx)
+        return;
+    try {
+      ctx->set_ca_bundle_path(path);
```

Comment:
Given that the set_ca_bundle_path() takes a std::string, care should be taken if path is nullptr, since this is undefined behaviour to instanciated a std::string from NULL. So:
```suggestion
      ctx->set_ca_bundle_path(path != nullptr ? path : "");
```

Reply:
Thanks :+1: